packages/cli/lang/en.lyaml

```diff
@@ -1115,6 +1119,8 @@ en:
           errors:
             buildIdDoesNotExist: "Build {{ buildId }} does not exist for project {{ projectName }}."
             buildAlreadyDeployed: "Build {{ buildId }} is already deployed."
+        overwritePrompt:
+          message: "Are you sure you want to overwrite {{ filePath }} on HubSpot?"
```

Comment:
Would it clearer to say "in your HubSpot account #{{accountId}}"?

Reply:
I do think we need to specify in the language that this is the remote file -- So, this suggestion makes sense to me. 👍

Reply:
If we refactor the option to prompt once at the start of the command -- then we can probably change this to "Are you sure you want to overwrite all files at `{{ filePath }}` in your HubSpot account `#{{ accountID }}`?